docs/topics/jvm/jvm-test-using-junit.md

```diff
@@ -0,0 +1,127 @@
+
+This tutorial shows you how to write a simple unit test and run it with the Gradle build tool.
+
+The example has the [kotlin.test](/api/latest/kotlin.test/index.html) library under the hood and runs the test using JUnit.
+
+To get started, first download and install the latest version of [IntelliJ IDEA](http://www.jetbrains.com/idea/download/index.html).
+
+## Add dependencies
+
+1. Open a Kotlin project in IntelliJ IDEA. If you don't have a project, [create one](/docs/tutorials/jvm-get-started.html#create-an-application).
+
+   > Specify **JUnit 4** as your test framework when creating a project.
+   >
+   {type="note"} 
+
+2. Open the `build.gradle(.kts)` file and add the following dependencies to the Gradle configuration to work with `kotlin.test` and `JUnit`:
+
+<tabs>
+
+   ```groovy
+   dependencies {
+       // Other dependencies.
+       // ...
+       testImplementation "org.jetbrains.kotlin:kotlin-test:1.3.11"
+       testImplementation "org.jetbrains.kotlin:kotlin-test-junit:1.3.11"
+   }
+   ```
+
+   ```kotlin
+   dependencies {
+       // Other dependencies.
+       // ...
+       testImplementation(kotlin("test"))
+       testImplementation(kotlin("test-junit"))
+   }
+   ```
+
+</tabs>
+
+## Add code to test
+
+1. Open the `main.kt` file in `src/main/kotlin`.
+
+   The `src` directory contains Kotlin source files and resources. The `main.kt` file contains sample code that will print `Hello World!`.
+
+2. Create the `Sample` class with the `sum()` function that sums two integers:
+
+```kotlin
+class Sample() {
+
+    fun sum(a: Int, b: Int): Int {
+        return a + b
+    }
+}
+```
+
+## Create a test
+
+1. In IntelliJ IDEA, select **Code** \| **Generate** \| **Test...** of the `Sample` class.
+
+   ![Create a test](create-test.png)
+
+2. Specify the name of the test class. For example, `SampleTest`.
+
+   IDEA creates the `SampleTest.kt` file in the `test` directory. It contains Kotlin test source files and resources.
+
+   > You can also create a `*.kt` file for test in `src/test/kotlin` manually.
+   >
+   {type="note"}
+
+2. Add the test code that for the `sum()` function in `SampleTest.kt`:
+   
+   * Define the test `testSum()` function using the [@Test annotation](/api/latest/kotlin.test/kotlin.test/-test/index.html).
+   * Check that the `sum()` function returns the value that is equal to the expected one using the [assertEquals()](/api/latest/kotlin.test/kotlin.test/-test/assert-equals.html) function.
+
+   ```kotlin
+   import kotlin.test.Test
+   import kotlin.test.assertEquals
+   
+   internal class SampleTest {
+   
+       private val classForTesting: Sample = Sample()
+   
+       @Test
+       fun testSum() {
+           val expected = 42
+           assertEquals(expected, classForTesting.sum(40, 2))
+       }
+   }
+   ```
+
+## Run a test
+
+1. Run the test using the gutter icon. You can also run a test via the command-line interface using the `./gradlew` command.
```

Comment:
> You can also run a test via the command-line interface using the `./gradlew` command

I suppose using the `./gradlew check` to execute the `check` task which itself depends on tasks running project tests.
Then, to run a single test, some additional arguments may be required. So may be better to say "You can also run all project tests ..."

Reply:
Yeah, thanks!
I've taken your suggestion and update the command to `./gradlew test` (according to official DSL)

Reply:
In a JVM project, yes. In an MPP project there is no `test` task, test tasks there have a platform prefix like `jvmTest`, `jsTest` etc, and there is a unifying `check` task that runs them all.

Reply:
We have another doc page on running test in MPP, but I'll update the command, thanks for the details!